rpcs3/Emu/Cell/Modules/sceNpTrophy.cpp

```diff
@@ -616,6 +549,103 @@ error_code sceNpTrophyGetTrophyInfo(u32 context, u32 handle, s32 trophyId, vm::p
 	return CELL_OK;
 }
 
+error_code sceNpTrophyUnlockTrophy(u32 context, u32 handle, s32 trophyId, vm::ptr<u32> platinumId)
```

Comment:
Don't move functions as possible. Otherwise it could be merged with conflicts fixed.

Reply:
It can't be helped.

Reply:
What? You kill all the change history by moving them, when they are almost identical.